src/app/visual/radio/radio-visual.component.html

```diff
@@ -81,3 +81,9 @@
     </ul>
   </div>
 </form>
+
+<button
+  type="button"
+  (click)="changeValue()">
```

Comment:
I don't see this function defined

Reply:
good catch! it was testing remnants. Removed.